CHANGELOG.md

```diff
@@ -39,6 +39,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [v4.0.3](https://github.com/Stride-Labs/stride/releases/tag/v4.0.0) - 2022-12-05
```

Comment:
Thanks for adding these. Should we be adding them with subsequent PRs, or all at once with a release like this? I think we agreed on the latter, but I'd like to make sure. The work to add these shouldn't all fall on you.

Reply:
I think we agreed on doing them with a release! It's pretty quick to go through them all at once, plus you need the commit hash for each bullet which you don't get until after branch is merged in